XamlStyler.Package/StylerPackage.cs

```diff
@@ -201,6 +209,32 @@ private void Execute(Document document)
             }
         }
 
+        private string GetConfigPathForItem(string path)
+        {
+            try
+            {
+                if (string.IsNullOrWhiteSpace(path))
+                {
+                    return null;
+                }
+
+                while ((path = Path.GetDirectoryName(path)) != null)
```

Comment:
Yeah, I think solution folder should suffice. Having to copy the settings file to each solution may be a brief chore, but it will at least make the solution owners think about it and to make sure they are aware of where the settings are coming from (if that makes sense)